bot/commands.js

```diff
@@ -233,10 +233,12 @@ const saveUserReview = async (targetUser, rating) => {
 const cancelAddInvoice = async (ctx, order, job) => {
   try {
     let userAction = false;
+    let userTelId = false;
```

Comment:
@Catrya tel means telegram? I dont think there's need to do abbreviations in variable names, they add confusion

Reply:
Hi @knocte, yes, Tel means Telegram. I can change it to userTelegramId to make it clearer, what do you think?

Reply:
sure, but then there's inconsistency because I think it's called userTgId in other places?

Reply:
Hi @knocte I named it `userTgId` because you didn't answer when I asked, and that name seemed better to me.